src/app/Table.elm

```diff
@@ -0,0 +1,29 @@
+module Table exposing (view, init, Msg)
```

Comment:
I agree... most things that other frameworks or languages use components for can be accomplished with a simple function in Elm, and it might be that the `Table` module doesn't end up needing an `update` function or its own `Model` type at all.